identity-account/src/account/builder.rs

```diff
@@ -60,10 +65,22 @@ impl AccountBuilder {
     self
   }
 
+  /// Apply configuration to the IOTA Tangle client for the given `Network`.
+  pub fn client<F>(mut self, network: Network, f: F) -> Self
+  where
+    F: Fn(ClientBuilder) -> ClientBuilder,
```

Comment:
I think the more appropriate bound would be `FnOnce`, as we don't call `f` repeatedly.